tools/klockstat.py

```diff
@@ -427,10 +435,10 @@ def display(sort, maxs, totals, counts):
 
 b = BPF(text=program)
 
-if not is_support_kfunc:
-    b.attach_kprobe(event="mutex_unlock",  fn_name="mutex_unlock_enter")
-    b.attach_kretprobe(event="mutex_lock", fn_name="mutex_lock_return")
-    b.attach_kprobe(event="mutex_lock",    fn_name="mutex_lock_enter")
+if not is_support_kfunc and kprobe_mutex_lock_func:
```

Comment:
I tried to look around at surrounding usages of `get_kprobe_functions` to take my clue about error reporting. From what I could tell most of them just attach to whatever `get_kprobe_functions` finds and if it doesn't find anything, they just silently don't attach.

So that's what I implemented here. But I'm definitely willing to add error reporting if that would be better received.